daemon/response.go

```diff
@@ -150,7 +151,7 @@ func ErrorResponse(status int) ErrorResponseFunc {
 		Status: status,
 	}
 
-	return r.SetError
+	return r.setError
```

Comment:
didn't come from this branch but I find it strange that ErrorResponse follow the *Response naming pattern but doesn't actually produce a response